exercises/concept/party-robot/party_robot_test.go

```diff
@@ -60,10 +65,10 @@ func TestAssignTable(t *testing.T) {
 			description:   "Greet the guest and give them directions to their seat",
 			name:          "Christiane",
 			direction:     "on the left",
-			tableNumber:   27,
+			tableNumber:   tableNumber,
 			distance:      23.7834298,
 			neighbourName: "Frank",
-			want:          "Welcome to my party, Christiane!\nYou have been assigned to table 1B. Your table is on the left, exactly 23.8 meters from here.\nYou will be sitting next to Frank.",
+			want:          fmt.Sprintf("Welcome to my party, Christiane!\nYou have been assigned to table %03d. Your table is on the left, exactly 23.8 meters from here.\nYou will be sitting next to Frank.", tableNumber),
```

Comment:
It feels a little strange that we are using the same way here to test the result than to build it. Additionally you fixed the issue that hard coding the table number does not work anymore but hard coding all other values still works.

I would recommend instead of using the random number, we add another test case to the table.  

I saw the issue that there is only one test case also exist for all other tests.
Would you be able to extend this PR to add a second test case to all tests AND make sure all test cases are different from the example shown in the instructions.md file? That would be very helpful.